app/src/main/java/de/danoeh/antennapod/fragment/NavDrawerFragment.java

```diff
@@ -91,6 +94,9 @@ public View onCreateView(@NonNull LayoutInflater inflater, @Nullable ViewGroup c
                 startActivity(new Intent(getActivity(), PreferenceActivity.class)));
         getContext().getSharedPreferences(PREF_NAME, Context.MODE_PRIVATE)
                 .registerOnSharedPreferenceChangeListener(this);
+
+        progressBar = root.findViewById(R.id.progressBar);
+        progressBar.bringToFront();
```

Comment:
Instead of calling this, please move the progress bar further down in the layout xml file.